### PR TITLE
Translate contribute/new-content/overview/ in Korean

### LINKED ISSUE #20752: Translate contribute/new-content/overview/ in Korean
**This is a Feature Request**

**What would you like to be added**

Translate contribute/new-content/overview/ in Korean.

**Why is this needed**

There is no document of contribute/new-content/overview/ which translated into Korean.

**Comments**

Page to translate: https://kubernetes.io/docs/contribute/new-content/overview/

/language ko

### PR DESCRIPTION
**Pull request by pjhwa**
This fixes #20752 .

This PR contains the following files which translated into Korean.
- content/ko/docs/contribute/new-content/_index.md
- content/ko/docs/contribute/new-content/overview.md

### CODE REVIEW

**Comment by pjhwa** [comment]
Deploy preview: https://deploy-preview-20754--k8s-dev-ko.netlify.app/ko/docs/contribute/new-content/overview/

**Comment by pjhwa** [comment]
> - 여러 언어로 문서화에 기여하거나 새로운 번역을 시작하는 방법에 대한 자세한 내용은 [현지화](/docs/contribute/localization)를 참고한다.

Line 25의 `현지화` 에 `한글화 가이드` 를 링크하는 것이 좋겠죠?
https://kubernetes.io/ko/docs/contribute/localization_ko/

File: content/ko/docs/contribute/new-content/overview.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+이 섹션에는 새로운 콘텐츠를 기여하기 전에 알아야 할 정보가 있다.
+
+
+{{% /capture %}}
+
+{{% capture body %}}
+
+## 기여하기에 대한 기본
+
+- 마크다운(markdown)으로 쿠버네티스 문서를 작성하고 [Hugo](https://gohugo.io/)를 사용하여 쿠버네티스 사이트를 구축한다.
```
Comment (sftim) [inline content/ko/docs/contribute/new-content/overview.md:20]: nit (OK to merge without fixing) / might not be a bug:
```suggestion
- 마크다운(Markdown)으로 쿠버네티스 문서를 작성하고 [Hugo](https://gohugo.io/)를 사용하여 쿠버네티스 사이트를 구축한다.
```

File: content/ko/docs/contribute/new-content/overview.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+- 소스는 [GitHub](https://github.com/kubernetes/website)에 있다. 쿠버네티스 문서는 `/content/en/docs/` 에서 찾을 수 있다. 일부 참조 문서는 `update-imported-docs/` 디렉터리의 스크립트에서 자동으로 생성된다.
```
Comment (sftim) [inline content/ko/docs/contribute/new-content/overview.md:21]:
```suggestion
- 소스는 [GitHub](https://github.com/kubernetes/website)에 있다. 쿠버네티스 문서는 `/content/ko/docs/` 에서 찾을 수 있다. 일부 참조 문서는 `update-imported-docs/` 디렉터리의 스크립트에서 자동으로 생성된다.
```
Alternatively, mention both the English originals and the localized Korean versions.

File: content/ko/docs/contribute/new-content/overview.md (hunk 1 above)
Comment (pjhwa) [reply content/ko/docs/contribute/new-content/overview.md:20]: Miss-typed. :-)
I'll change it.

File: content/ko/docs/contribute/new-content/overview.md (hunk 2 above)
Comment (pjhwa) [reply content/ko/docs/contribute/new-content/overview.md:21]: Yes, I also missed this point. I'll change it, too.

**Comment by pjhwa** [comment]
@sftim Thank you for your review. 
I've changed the document as your comments.

File: content/ko/docs/contribute/new-content/overview.md [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+{{% capture body %}}
+
+## 기여하기에 대한 기본
+
+- 마크다운(Markdown)으로 쿠버네티스 문서를 작성하고 [Hugo](https://gohugo.io/)를 사용하여 쿠버네티스 사이트를 구축한다.
+- 소스는 [GitHub](https://github.com/kubernetes/website)에 있다. 쿠버네티스 문서는 `/content/en/docs/` 에서 찾을 수 있다. 일부 참조 문서는 `update-imported-docs/` 디렉터리의 스크립트에서 자동으로 생성된다.
+- [페이지 템플릿](/docs/contribute/style/page-templates/)은 Hugo에서 문서 콘텐츠의 프리젠테이션을 제어한다.
+- 표준 Hugo 단축코드(shortcode) 이외에도 설명서에서 여러 [사용자 정의 Hugo 단축코드](/docs/contribute/style/hugo-shortcodes/)를 사용하여 콘텐츠 표시를 제어한다.
+- 문서 소스는 `/content/` 에서 여러 언어로 제공된다. 각 언어는 [ISO 639-1 표준](https://www.loc.gov/standards/iso639-2/php/code_list.php)에 의해 결정된 2문자 코드가 있는 자체 폴더가 있다. 예를 들어, 영어 문서의 소스는 `/content/ko/docs/` 에 저장된다.
+- 여러 언어로 문서화에 기여하거나 새로운 번역을 시작하는 방법에 대한 자세한 내용은 [현지화](/ko/docs/contribute/localization_ko/)를 참고한다.
```
Comment (sftim) [inline content/ko/docs/contribute/new-content/overview.md:25]: _영어_? This still doesn't seem right @pjhwa (sorry - I should have been more clear).

Comment (pjhwa) [reply content/ko/docs/contribute/new-content/overview.md]: Oops! I've just changed '영어' to '한글'. Thank you.

File: content/ko/docs/contribute/new-content/overview.md [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+## 기여하기에 대한 기본
+
+- 마크다운(Markdown)으로 쿠버네티스 문서를 작성하고 [Hugo](https://gohugo.io/)를 사용하여 쿠버네티스 사이트를 구축한다.
+- 소스는 [GitHub](https://github.com/kubernetes/website)에 있다. 쿠버네티스 문서는 `/content/ko/docs/` 에서 찾을 수 있다. 일부 참조 문서는 `update-imported-docs/` 디렉터리의 스크립트에서 자동으로 생성된다.
+- [페이지 템플릿](/docs/contribute/style/page-templates/)은 Hugo에서 문서 콘텐츠의 프리젠테이션을 제어한다.
+- 표준 Hugo 단축코드(shortcode) 이외에도 설명서에서 여러 [사용자 정의 Hugo 단축코드](/docs/contribute/style/hugo-shortcodes/)를 사용하여 콘텐츠 표시를 제어한다.
+- 문서 소스는 `/content/` 에서 여러 언어로 제공된다. 각 언어는 [ISO 639-1 표준](https://www.loc.gov/standards/iso639-2/php/code_list.php)에 의해 결정된 2문자 코드가 있는 자체 폴더가 있다. 예를 들어, 한글 문서의 소스는 `/content/ko/docs/` 에 저장된다.
+- 여러 언어로 문서화에 기여하거나 새로운 번역을 시작하는 방법에 대한 자세한 내용은 [현지화](/ko/docs/contribute/localization_ko/)를 참고한다.
+
+
```
Comment (ClaudiaJKang) [inline content/ko/docs/contribute/new-content/overview.md:27]:
```suggestion
```

- [ ] 원문과 개행이 다릅니다.

File: content/ko/docs/contribute/new-content/overview.md [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+:---------|:------------
+현재 릴리스의 기존 또는 새로운 영어 콘텐츠 | `master`
+기능 변경 릴리스의 콘텐츠 | `dev-release-<version>` 패턴을 사용하여 기능 변경이 있는 주 버전과 부 버전에 해당하는 브랜치. 예를 들어, `{{< latest-version >}}` 에서 기능이 변경된 경우, ``dev-{{< release-branch >}}`` 에 문서 변경을 추가한다.
+다른 언어로된 콘텐츠(현지화) | 현지화 규칙을 사용. 자세한 내용은 [현지화 브랜치 전략](/docs/contribute/localization/#branching-strategy)을 참고한다.
+
+
+어떤 브랜치를 선택해야 할지 잘 모르는 경우 슬랙의 `#sig-docs` 에 문의한다.
+
+{{< note >}}
+풀 리퀘스트를 이미 제출했는데 기본 브랜치가 잘못되었다는 것을 알게 되면, 제출자(제출자인 여러분만)가 이를 변경할 수 있다.
```
Comment (ClaudiaJKang) [inline content/ko/docs/contribute/new-content/overview.md:50]: > origin
> If you already submitted your pull request and you know that the base branch	
> was wrong, you (and only you, the submitter) can change it.	

```suggestion
풀 리퀘스트를 이미 제출했는데 기본 브랜치가 잘못되었다는 것을 알게 되면, 
제출자(제출자인 여러분만)가 이를 변경할 수 있다.
```

- [ ] 원문의 개행 유지가 필요합니다.

File: content/ko/docs/contribute/new-content/overview.md (hunk 4 above)
Comment (pjhwa) [reply content/ko/docs/contribute/new-content/overview.md:27]: 반영하겠습니다.

File: content/ko/docs/contribute/new-content/overview.md (hunk 5 above)
Comment (pjhwa) [reply content/ko/docs/contribute/new-content/overview.md:50]: 반영하겠습니다.

**Comment by pjhwa** [comment]
@ClaudiaJKang 리뷰 감사합니다!
의견 주신 내용은 모두 반영하여 업데이트하고, 스쿼시 처리하겠습니다.

**Comment by ClaudiaJKang** [comment]
@pjhwa 감사합니다.

/lgtm
/approve

**Pull request merged**